### PR TITLE
CONTRIBUTING.rst: nosetests --> pytest

### PR DESCRIPTION
**Pull request by cclauss**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to the dependencies
- [ ] I have updated the dependencies accordingly
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.rst
Comment (Pierre-Sassoulas) [inline CONTRIBUTING.rst:40]: We're still targeting 3.6, for now, see https://github.com/PyCQA/prospector/pull/511#issuecomment-1156178453

**Review by Pierre-Sassoulas — APPROVED** [review]
Nice catch. Blocked by #511

File: CONTRIBUTING.rst
Comment (cclauss) [reply CONTRIBUTING.rst:40]: My guess is that you and I are the only ones that will read this file between not and the dropping of py36 and this pull request demonstrates that this file is updated as an afterthought.
https://devguide.python.org/versions/#versions

Comment (Pierre-Sassoulas) [reply CONTRIBUTING.rst]: I wanted to wait for 511 to be merged to merge this one just after (and keep your original change) considering the cache issue. But let's merge if the cache issue is fixed. It could be due to a new interpreter being released so the cache is confused about it.

**Pull request closed without merging**
